src/views/pages/network-providers.html

```
@@ -52,7 +52,7 @@ <h1>
 				noDataText="No providers found. Please try again later."
 			/>
 
-			{{#if topProviders.length}}
+			{{#if topProviders.length >= 3}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)